storybook/stories/ServerItem.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions, react/prop-types */
+import React from 'react';
+import { SafeAreaView, FlatList } from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+
+import ServerItem from '../../app/presentation/ServerItem';
+import { ThemeContext } from '../../app/theme';
+
+const stories = storiesOf('ServerItem', module);
+
+const themes = {
+	light: 'light',
+	dark: 'dark',
+	black: 'black'
+};
+
+const item = {
+	name: 'https://open.rocket.chat/',
+	id: 'some.user',
+	iconURL: 'https://open.rocket.chat/images/logo/android-chrome-512x512.png'
+};
+
+const item2 = {
+	name: 'https://superlongservername.rocket.chat/',
+	id: 'superlongusername.tologintoasuperlongservername'
+};
+
+stories.add('unselected server', () => (
+	<ServerItem
+		item={item}
+		onPress={() => alert('Hi')}
+		onLongPress={() => alert('Deleted')}
+		theme={themes.light}
+	/>
+));
+
+stories.add('selected server', () => (
+	<ServerItem
```

Comment:
Create a basic component story and then overload props according to what you're testing.